docker/base/nginx.conf

```diff
@@ -1,7 +1,7 @@
 worker_processes  1;
 
 error_log  /var/log/nginx/error.log warn;
-pid        /var/run/nginx/nginx.pid;
+pid        /tmp/nginx.pid;
```

Comment:
I don't think we need to change this

Reply:
We have, for dev instance - I'm getting permission errors

Reply:
Then we should fix the dev configuration, not do a workaround in prod

Reply:
See e09d1295c5da0f936b8202c62236a15d4884c402